util/secret/certificates.go

```diff
@@ -395,6 +422,17 @@ func generateCACert() (*certs.KeyPair, error) {
 	}, nil
 }
 
+func generateClientCert(cn string, ca *certs.KeyPair) (*certs.KeyPair, error) {
```

Comment:
these arguments are unused, either replace the names with `_` or remove the arguments entirely in order to clarify intent